core/src/main/java/com/hcl/ecomm/core/servlets/GetWishListItemsServlet.java

```diff
@@ -64,4 +70,8 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         }
 
     }
+    public JSONObject getWishListItems(String customerToken) throws JSONException {
+        JSONObject jo2 = new JSONObject(wishListService.getWishListItems(customerToken).toString());
```

Comment:
Why is this JSONObject created? @uditnaagar777

Reply:
wishListService.getWishListItems() returns JsonObject, So to convert jsonObject to JSONObject.
We are using "JSONObject .getJSONArray()" which is not available in case of JsonObject hence the conversion is required.

Reply:
I meant why not directly return "wishListService.getWishListItems(customerToken)".

Reply:
The return type "wishListService.getWishListItems()" is JsonObject but we have to use "getJSONArray()" which is available in  JSONObject & not available in JsonObject.